app/src/main/res/navigation/nav_graph.xml

```diff
@@ -1186,21 +1186,21 @@
     </navigation>
 
     <navigation
-        android:id="@+id/credit_cards_graph"
-        app:startDestination="@id/creditCardsSettingFragment">
+        android:id="@+id/autofill_graph"
+        app:startDestination="@id/autofillSettingFragment">
         <fragment
-            android:id="@+id/creditCardsSettingFragment"
-            android:name="org.mozilla.fenix.settings.creditcards.CreditCardsSettingFragment"
+            android:id="@+id/autofillSettingFragment"
+            android:name="org.mozilla.fenix.settings.autofill.AutofillSettingFragment"
             android:label="@string/preferences_credit_cards">
```

Comment:
Are we waiting to adjust the label?

Reply:
Ya, I figure I would do this when we make the official switch over.